### DISCUSSION TITLE
Welcome to Labrea Discussions!

### DISCUSSION THREAD

**Discussion by austinwarner-8451** [8451/labrea#20] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with Labrea users, especially anyone outside of 84.51º. 
  * [🙏 Q&A](https://github.com/8451/labrea/discussions/new?category=q-a): If you have a question about Labrea
  * [💡 Ideas](https://github.com/8451/labrea/discussions/new?category=ideas): If you have an idea for a new Labrea feature, and want to discuss before opening an issue
  * [💬 General](https://github.com/8451/labrea/discussions/new?category=general): For other miscellaneous conversation about Labrea

**Comment by betsyehmcke-8451** [comment]
Thank you, Austin! LOL
